nucleus/payara-modules/notification-core/src/main/java/fish/payara/nucleus/notification/log/LogNotificationEventFactory.java

```diff
@@ -90,4 +90,12 @@ public LogNotificationEvent buildNotificationEvent(String name, List<HealthCheck
 
         return notificationEvent;
     }
+
+    @Override
+    public LogNotificationEvent buildNotificationEvent(String subject, String message) {
```

Comment:
Is it possible for this method to overload one of the others or vica versa?

Reply:
Should be possible, but just used the code from Payar 5 so future changes can easily be backported.

Reply:
That makes sense, cheers!